internal/config/config.go

```diff
@@ -78,6 +78,9 @@ type Complement struct {
 	CAPrivateKey  *rsa.PrivateKey
 
 	BestEffort bool
+
+	// The hostname of Complement from the perspective of a Homeserver running inside a container
+	HostnameRunningComplement string
```

Comment:
Please update the docstring to match other env vars in this struct. Use this format so we can automatically generate docs:
```go
// Name: COMPLEMENT_HOSTNAME_RUNNING_COMPLEMENT
// Default: host.docker.internal
// Description: Some useful description here,
// multi-line is fine.
```

Please be thorough and explain:
 - Why you might want to set this.
 - What value you might want to set this to in Podman as an example.